exercises/triangle/src/example.h

```diff
@@ -0,0 +1,13 @@
+#ifndef TRIANGLE_H
+#define TRIANGLE_H
+
+enum flavor {
```

Comment:
To stay consistent with other exercises (since this will be public) copy the enum formatting used for the Allergies exercise.

So use a typedef instead, rename `flavor` to `Flavor_t`, and capitalize the values (`invalid` -> `Invalid`)